Cart Flow/Custom Views/Labels/CFBodyLabel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CFBodyLabel.swift
+//  Cart Flow
+//
+//  Created by Scott Quintana on 8/6/20.
+//  Copyright © 2020 Scott Quintana. All rights reserved.
+//
+
+import UIKit
+
+class CFBodyLabel: UILabel {
+
+    override init(frame: CGRect) {
+       super.init(frame: frame)
+       configure()
+   }
+   
+   
+   required init?(coder: NSCoder) {
+       fatalError("init(coder:) has not been implemented")
+   }
+   
+   
+   convenience init(textAlignment: NSTextAlignment) {
+       self.init(frame: .zero)
+       self.textAlignment = textAlignment
+   }
+   
+   
+   private func configure() {
+       textColor = .label
+       font = UIFont.preferredFont(forTextStyle: .body)
+       adjustsFontForContentSizeCategory = true
+       
+       adjustsFontSizeToFitWidth = true
+       minimumScaleFactor = 0.75
```

Comment:
Yay `minimumScaleFactor`. This is one of my favorite features of UIKit.

Reply:
I've had one little problem with it, and that's when something does shrink to fit, it loses its centerY alignment. Say, a label on a button, when it shrinks it's now centered slightly lower on the button. Tried setting the baseline adjustment to .center and didn't seem to fix it